examples/one_degree_simulation.jl

```diff
@@ -20,25 +20,28 @@ using ClimaOcean.ECCO: download_dataset
 
 # ### ECCO files
 
-dates = DateTime(1993, 1, 1) : Month(1) : DateTime(1994, 1, 1)
+start_date = DateTime(1993, 1, 1) 
+stop_date = DateTime(1993, 12, 1) 
+dates = range(start_date, step=Month(1), stop=stop_date)
 ecco_temperature = Metadata(:temperature; dates, dataset=ECCO4Monthly())
-ecco_salinity    = Metadata(:salinity;    dates, dataset=ECCO4Monthly())
+ecco_salinity = Metadata(:salinity; dates, dataset=ECCO4Monthly())
 
 download_dataset(ecco_temperature)
 download_dataset(ecco_salinity)
 
 # ### Grid and Bathymetry
 
 arch = CPU()
-Nx = 256
-Ny = 128
+Nx = 360
+Ny = 170
 Nz = 40
 
 r_faces = exponential_z_faces(; Nz, depth=4000, h=34)
 underlying_grid = TripolarGrid(arch; size = (Nx, Ny, Nz), z = r_faces, halo = (5, 5, 4))
 
 ## 75 interpolation passes smooth the bathymetry near Florida so that the Gulf Stream is able to flow:
```

Comment:
the comment says 75 but we do 10

Reply:
ill change it back to 75 when we are tready to merge. It's just expensive for testing.